pom.xml

```diff
@@ -73,7 +73,10 @@
                 <version>2.3.4</version>
                 <configuration>
                     <slingUrl>${sling.url}</slingUrl>
-                    <password>ccsadmin</password>
+                    <user>admin</user>
+                    <password>admin</password>
```

Comment:
Would you mind changing this to our non-default ccsadmin? Or, I wonder if we could handle this a bit better, perhaps using the profiles that we talked about earlier today. Certainly open to suggestions.

Reply:
Done. Removed the explicit defaults (they are still defaults per the plugin). For now the way to customize this will be to provide the `-Dsling.user` and `-Dsling.password` properties if they differ.